universal-login-commons/src/core/constants/test.ts

```diff
@@ -3,6 +3,10 @@ import {ETHER_NATIVE_TOKEN} from './constants';
 
 export const TEST_ACCOUNT_ADDRESS = '0x0000000000000000000000000000000000000001';
 
+export const TEST_KEY = '0x5000000000000000000000000000000000000005';
```

Comment:
We should stick to the names convention and name it `TEST_ACCOUNT_ADDRESS_1`, `TEST_ACCOUNT_ADDRESS_2`. 
Btw. I don't feel good with `TEST_KEY`. `TEST_PUBLIC_KEY` would be better.

Reply:
Semantically that are two different things. Just for self-documenting test, we put Key, where we mean ephemeral key and account when we mean ethereum account.